ql/src/test/results/clientpositive/alter_tableprops_numeric.q.out

```diff
@@ -0,0 +1,52 @@
+PREHOOK: query: create external table stat_test (a int)
+PREHOOK: type: CREATETABLE
+PREHOOK: Output: database:default
+PREHOOK: Output: default@stat_test
+POSTHOOK: query: create external table stat_test (a int)
+POSTHOOK: type: CREATETABLE
+POSTHOOK: Output: database:default
+POSTHOOK: Output: default@stat_test
+PREHOOK: query: alter table stat_test set TBLPROPERTIES('numFiles'='1', 'numRows'='2', 'totalSize'='3', 'rawDataSize'='4', 'numFilesErasureCoded'='5', 'STATS_GENERATED_VIA_STATS_TASK'='true')
+PREHOOK: type: ALTERTABLE_PROPERTIES
+PREHOOK: Input: default@stat_test
+PREHOOK: Output: default@stat_test
+POSTHOOK: query: alter table stat_test set TBLPROPERTIES('numFiles'='1', 'numRows'='2', 'totalSize'='3', 'rawDataSize'='4', 'numFilesErasureCoded'='5', 'STATS_GENERATED_VIA_STATS_TASK'='true')
+POSTHOOK: type: ALTERTABLE_PROPERTIES
+POSTHOOK: Input: default@stat_test
+POSTHOOK: Output: default@stat_test
+PREHOOK: query: describe formatted stat_test
+PREHOOK: type: DESCTABLE
+PREHOOK: Input: default@stat_test
+POSTHOOK: query: describe formatted stat_test
+POSTHOOK: type: DESCTABLE
+POSTHOOK: Input: default@stat_test
+# col_name            	data_type           	comment             
+a                   	int                 	                    
+	 	 
+# Detailed Table Information	 	 
+Database:           	default             	 
+#### A masked pattern was here ####
+Retention:          	0                   	 
+#### A masked pattern was here ####
+Table Type:         	EXTERNAL_TABLE      	 
+Table Parameters:	 	 
+	EXTERNAL            	TRUE                
+	STATS_GENERATED_VIA_STATS_TASK	true                
+	bucketing_version   	2                   
+#### A masked pattern was here ####
+	numFiles            	0                   
```

Comment:
I wonder why this is not 1

Reply:
Yeah, I was wondering about that too, similarly the "totalSize" also cannot be altered directly. I do not know if this is intentional or not, I did not dig myself into it.